tests/model_fields/tests.py

```diff
@@ -661,6 +689,15 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+            except (DataError, IntegrityError):
+                pass  # backends such as Postgres don't fail silently
+            else:
+                qs = self.model.objects.filter(value__lte=min_value)
```

Comment:
Code looks good. Regarding the name, I'd like to make it clear when the truncation happens. For example, I could imagine truncation of decimal places (maybe that happens too). I'm certainly open to other suggestions. We could even add 'silently' to the name. Since no other database (probably) exhibits this behavior, I think it's fine to make it as descriptive as possible.